go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	github.com/coreos/etcd v3.3.13+incompatible
 	github.com/fatih/color v1.7.0
+	github.com/gogo/protobuf v1.2.1
```

Comment:
dony use `gogo/protobuf`. Its unstable for now for marshalling and unmarshalling. Instead use `golang/protobuf`. Remove this import from go mod and also from replace in any of your file with `golang/protobuf`